Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM debian:stable-slim
 
-ENV TERRAFORM_VERSION 0.11.7
+ENV TERRAFORM_VERSION 0.11.11
```

Comment:
Why we need update the terraform version?

Reply:
Terraform AWS Provider requires 0.10+ version.

Reply:
Sorry @lborguetti, I made a mistake. Terraform AWS Provider requires terraform 0.10+, so version 0.11.7 is enough. I've already fixed it in this commit: [https://github.com/NeowayLabs/packer-images/pull/22/commits/414cb93700742399e2b590fd708135ba89889008](414cb93).